lib/ClangImporter/ClangImporter.cpp

```diff
@@ -2889,3 +2889,15 @@ void ClangImporter::Implementation::dumpSwiftLookupTables() {
   llvm::errs() << "<<Bridging header lookup table>>\n";
   BridgingHeaderLookupTable->dump();
 }
+
+DeclName ClangImporter::
+analyzeImportedName(const clang::NamedDecl *ClangDecl,
+                    StringRef BaseName, ArrayRef<StringRef> SelectorPieces,
+                    version::Version version) {
+  swift::importer::PreferredName Name(BaseName, SelectorPieces);
```

Comment:
Nitpick: This is a context that has at least one of these namespaces included; please omit the qualification.